app/controllers/landlords_controller.rb

```diff
@@ -1,6 +1,8 @@
 class LandlordsController < ApplicationController
 
   def new
+    @landlord = Landlord.new
+    @user = User.new
```

Comment:
To play with devise (gem for user authentication) we will probably but this in one of the user controllers. We can build that together sometime as working with devise is kind of a pain.